server/routes/index.js

```diff
@@ -4,11 +4,13 @@ const authRouter = require('./auth');
 const feedBackRouter = require('./feedBackRouter');
 const profileRouter = require('./profile');
 const messageRouter = require('./message');
+const communityRouter = require('./community');
 
 router.use('/languages', languageRouter);
 router.use('/auth', authRouter);
 router.use('/feedback', feedBackRouter);
 router.use('/profile', profileRouter);
 router.use('/message', messageRouter);
+router.use(communityRouter);
```

Comment:
use prefix > before communityRouter
('/search', communityRouter)
....
and those in communityRouter :
('/', nameFilter)
('/', langFilter)

Reply:
@MohammedMahani , you can partition between filters on front-end and use a single controller for that